modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -36,244 +39,320 @@ from Canvas Kit v6 to v7:
+We are [hard-deprecating](#hard-deprecation) `PageHeader`. Please reference this
+[example](https://workday.github.io/canvas-kit/?path=/story/examples-pageheader-react--basic) to
+migrate away from `PageHeader`.
 
-### Header / GlobalHeader
+#### Header / Global Header
 
-We are [hard deprecating](#hard-deprecation) `Header`. If you would like to migrate away from this
-deprecated component now, you can reference
-[this example](https://workday.github.io/canvas-kit/?path=/story/examples-globalheader-react--basic)
+We are [hard-deprecating](#hard-deprecation) `Header`. Please reference this
+[example](https://workday.github.io/canvas-kit/?path=/story/examples-globalheader-react--basic) to
+migrate away from `Header`.
 
-## ActionBar Component Updates
+---
 
-We've refactored our ActionBar components to simplify logic.
+### Component Promotions
 
-PR: https://github.com/Workday/canvas-kit/pull/1396
+We've [promoted](https://github.com/Workday/canvas-kit/issues/1395) the following components in v7:
 
-**Changes:**
-
-- Component has been converted to compound component.
-- `fixed` prop has been removed from component.
-- Now ActionBar uses `position` prop to set container position instead `fixed` prop.
-- Fixed position has been set as default if there is not passed `position` prop.
-
-## Status Indicator Width
-
-Status Indicators currently truncate when they reach their max width of `150px`. After receiving
-requests to increase this, we have upped it by 25% to `200px`.
+- `Box` from `@workday/canvas-kit-labs/common` to `@workday/canvas-kit-react/layout`
+- `Flex` from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
+- `Stack` from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
 
-We've also added a prop (`maxWidth`) to configure this if you need to. While this increases
-flexibility, please keep in mind that status text should be short, direct, and preferably a single
-word.
+🤖 The codemod will update imports for these promoted components.
 
 ```tsx
-<StatusIndicator label="Slightly Longer Status" type={StatusIndicator.Type.Gray} maxWidth={250} />
+// v6
+import {Box} from '@workday/canvas-kit-labs-react/common';
+
+// v7
+import {Box} from '@workday/canvas-kit-react/layout';
 ```
 
-## Buttons
+Note that the APIs for these promoted components remain unchanged.
+
+---
 
-To consolidate button APIs, we removed `IconButton` in favor of `PrimaryButton`, `SecondaryButton`
-and `TertiaryButton`.
+### Action Bar
 
-### Breaking Changes
+We've updated `ActionBar` to use the
+[createComponent](/getting-started/for-developers/resources/creating-compound-components/#disclosurecontent-component)
+utility from our
+[common](https://github.com/Workday/canvas-kit/blob/ff77c5bd83e41c3ab2b9c55e41a8b7c1fde33a1b/modules/react/common/lib/utils/components.ts#L167)
+module; it now supports [ref forwarding](https://reactjs.org/docs/forwarding-refs.html) and using
+the `as` prop to change the rendered element.
```

Comment:
The migration guide originally stated that Action Bar had been [converted to a compound component](https://github.com/Workday/canvas-kit/blob/prerelease/major/modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx#actionbar-component-updates).

After discussing with @NicholasBoll, we determined that Action Bar does not qualify as a compound component (even though it uses `createComponent`) because it lacks subcomponents. That said, I do call out that Action Bar now supports ref forwarding and the `as` prop as a result of using `createComponent`.